### ISSUE TITLE
Create minimal verison without facial recognition

### ISSUE THREAD

**Issue by pangolingo** [ryanbateman/vision-bot#2]
I'm lazy and didn't want to set up an S3 bucket or configure the Heroku buildpack.

Can I suggest a "vision-without-facial.coffee" file that doesn't include the facial recognition stuff? Or possible a feature flag that you could set with an environment variable?

**Comment by ryanbateman** [comment]
Yes, it's something I'd like to do as well. It may be as simple as not doing the additional steps if you've not set up the S3 config, but that'd still include an additional library, I suppose. Would that additional requirement, that you don't necessarily actually use, be an issue for you?

**Comment by pangolingo** [comment]
Yes that makes sense rather than maintaining two files. Myself, I just commented out the S3, canvas, and faceAnnotations stuff.

**Comment by Jaglag** [comment]
++ for minimal version!
would be nice to have
